.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install ansible==2.2.2
+script:
+  - cd ansible && ansible-playbook --version
```

Comment:
Yes that is true, maybe we can, however, have a special vault with dummy values that is only used in CI and at least runs the syntax check.